src/main/scala/uk/gov/hmrc/domain/Nino.scala

```diff
@@ -18,13 +18,13 @@ package uk.gov.hmrc.domain
 
 import play.api.libs.json.{Reads, Writes}
 
-case class Nino(nino: String) extends TaxIdentifier with SimpleName {
-  require(Nino.isValid(nino), s"$nino is not a valid nino.")
-  override lazy val toString = nino
+case class Nino(override val value: String) extends TaxIdentifier with SimpleName {
```

Comment:
This breaks backwards-compatibility, the `nino` value has gone. Also, the `override`s are spurious. Suggest we keep this class as it was, but with just add the new `def formatted`